scipy/stats/_entropy.py

```diff
@@ -196,7 +257,60 @@ def differential_entropy(
     logs = np.log(
         len(differences) * differences / (2 * window_length),
     )
-    if base is not None:
-        logs /= np.log(base)
```

Comment:
@tupui There's no reason to divide all the terms individually;  might as well take the mean first, then divide a single number, right? Also, moving it out of this Vasicek-specific function allows the change of base to be shared by all the methods.

Is that right? Is conversion of entropy from one base ("units") to another as simple as that?

Reply:
Yes makes sense to do it afterwards.